### ISSUE TITLE
Compare DeepSpeech (w/Dropout) vs DeepSpeech (w/o Dropout) + FNN BatchNorm + RNN BatchNorm

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#375]

**Comment by andi4191** [comment]
@kdavis-mozilla : By FNN BatchNorm and RNN BatchNorm, are we referring to separate cases where:
1. FNN Batch Norm - Batch Norm applied over layers h1, h2, h3, h5, h6 and not on Bidirectional RNN layer
2. RNN Batch Norm - Batch Norm applied over Bidirectional RNN layer and not on h1, h2, h3, h5, h6

**Comment by kdavis-mozilla** [comment]
@andi4191 Correct

**Comment by kdavis-mozilla** [comment]
Closing for lack of activity.
